GOVERNANCE.md

```diff
@@ -18,15 +18,65 @@ This project may include the following roles. Additional roles may be adopted an
+
+**Recording.** Once a decision has been made during the consensus-seeking phase or by a formal vote, the initiative `sponsor` will record the decision in the related discussion, for posterity.
+
+### 3.2.2. Keys to Successful Consensus
+
+**Allow Time for Decisions**: While we want prioritize a lean and fast process, we also want to make sure that everyone express their opinions. The process should never feel "rushed."
+
+**Use Voting at the End of the Discussion**: Expanding on the item above, formal votes should be called only when discussion has explored the space, stakeholders have provided input and consensus hasn't been reached through the informal process. `Maintainers` should be should not call a vote to short-circuit an ongoing discussion that is still productive in terms of exploring ideas and feedback.
+
+**Give the right weight to decisions** `Maintainers` and those proposing decisions should explicitly distinguish between decisions that are two-way (easy to reverse later) and one-way (difficult or impossible to reverse later). For one-way doors, `maintainers` should carefully weigh alternatives and tradeoffs and take extra care to ensure broad participation and stakeholder input. For two-way doors, `maintainers` should feel free to move quickly, without compromising the principles and procedures described herein.
+
+**Utilize Data** In the context of the above item and in the interest of making solid decisions, arguments should bring data and examples to back them up. "Gut feelings" will not be considered with the same relevance of arguments backed by data and examples.
+
+**Acknowledge Some Questions Don't Have 'Right' Answers**: Expanding on the item above, many proposals or question won't have any objectively 'right' answers (e.g. naming things). `Maintainers` should acknowledge when they are expressing a preference, while avoiding declaring their preferences as "correct." Maintainers should also be wary that these matters can be the most prone to [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality) and judge how much time they're spending on these matters accordingly.
+
+**Remember How We Work** Given how passionate `maintainers` and users in general can be passionate about the project, discussion can get passionate. This should never be an excuse to not operate with all our principles in mind and make sure we treat each other with respect, empathy and kindness.
+
+**Favor Explicit over Implicit** `Sponsors` and `maintainers` should proactively solicit input from relevant stakeholders and should not assume that silence is consent without attempting to reach out to those individuals.
+
+**Plan before acting** If you are interested in a decision being made, it is your responsibility to encourage voting member/stakeholder/community participation in the decision-making process. If you cannot get such participation, you may want to hold off on doing any significant work on the matter.
```

Comment:
If people don't participate, in some cases that could be taken as a sign that most people just don't care? Either the decision won't affect them or they don't understand yet how it will affect them. In the case where it actually won't affect them, it's pretty reasonable to just go ahead with the engaged parties. In the case where they can't meaningfully participate because they don't understand the issues at hand and what their consequences are, then maybe it's important to address the situation in some way, maybe with more explanation.

Reply:
Yeah, that's a good point. I actually haven't thought of that aspect which is actually even important right now where we are still in the early phases of spreading the word and trying to get people involved. With that in mind, I'd propose to to remove this last sentence for the time being